cmd/minishift/cmd/start.go

```diff
@@ -406,8 +405,21 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 
 		fmt.Print("-- Starting Minishift VM ...")
 	} else {
+		s, err := sshutil.NewRawSSHClient(machineConfig.RemoteIPAddress, machineConfig.SSHKeyToConnectRemote, machineConfig.RemoteSSHUser)
+		if err != nil {
+			atexit.ExitWithMessage(1, fmt.Sprintf("Error creating ssh client: %v", err))
+		}
+		fmt.Printf("-- Preparing Remote Machine ...")
+		progressDots := progressdots.New()
+		progressDots.Start()
+		if err := generic.PrepareRemoteMachine(s); err != nil {
```

Comment:
... changing the package name makes more sense, as the responsibility is clear. Besides, 'generic' is an awkward name to begin with as we mean `remote` everywhere else anyway.

Reply:
I am not sure if putting `remote` everywhere is a good idea because someone might try to use his local machine to start OpenShift.  I also I agree that `generic` is also not a great word as it can confuse people.

Reply:
@praveenkumar @gbraad  I could not find any better name. lets go with `remotehost`.